packages/table/src/regions.ts

```diff
@@ -528,6 +528,20 @@ export class Regions {
         return regionGroups;
     }
 
+    public static regionArraysEqual(regionsA: IRegion[], regionsB: IRegion[]) {
+        if (regionsA === undefined && regionsB === undefined) {
+            return true;
+        } else if (regionsA === null && regionsB === null) {
```

Comment:
No great reason aside from the desire to use `===` everywhere for consistency. Strong aversion to keeping as is? I'm ambivalent.